starboard/android/shared/platform_configuration/BUILD.gn

```diff
@@ -23,6 +23,13 @@ config("platform_configuration") {
     "log",
     "mediandk",
   ]
+  cflags = [
+    "-fsigned-char",
```

Comment:
I'm interested to know what error you ran into here . 

This will get applied to all targets built with android chromium toolchain - so a bit worried about adding this

Reply:
might be good to add a TODO but that's all

Reply:
Created b/376519321